setup.cfg

```diff
@@ -1,10 +1,2 @@
-[nosetests]
-with-coverage = true
-cover-package = fakeldap
-cover-html = true
-cover-erase = true
-cover-inclusive = true
-nocapture=1
```

Comment:
We get rid of the coverage set up, so if we merge this, we should open an issue to set this up with pytest

We weren't enforcing coverage or properly printing any info about  it anyway, so it wasn't very visible to begin with though

Reply:
Sounds good!